rules/windows/process_creation/proc_creation_win_ads_stored_dll_execution_rundll32.yml

```
@@ -0,0 +1,22 @@
+title: Suspicious Alternate Data Stream Execution Via RunDLL32
+id: 9248c7e1-2bf3-4661-a22c-600a8040b446
+status: experimental
+description: Execution of DLL's stored in alternate data stream using RunDLL32
+references:
+    - https://lolbas-project.github.io/lolbas/Binaries/Rundll32
+author: Harjot Singh, '@cyb3rjy0t'
+date: 2023/01/11
+tags:
+    - attack.defense_evasion
+    - attack.t1564.004
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        Image|endswith: '\rundll32.exe'
+        CommandLine|re: '.+\:.+\..+\:.+\..+\,.+$'
```

```suggestion
        CommandLine|re: '\w:\\.+\..+:.+\.dll.*,\w+'
```
Use Rundll32.exe to execute a .DLL file stored in an Alternate Data Stream (ADS).
`\w:\\` -> C:\\
`.+\..+:` -> path\filename.extension: 
`.+\.dll` ->  file.dll
`.*,\w+` -> ",fonction